Userland/Services/AudioServer/main.cpp

```diff
@@ -16,23 +18,33 @@ ErrorOr<int> serenity_main(Main::Arguments)
     TRY(Core::System::pledge("stdio recvfd thread accept cpath rpath wpath unix"));
 
     auto config = TRY(Core::ConfigFile::open_for_app("Audio", Core::ConfigFile::AllowWriting::Yes));
-    TRY(Core::System::unveil(config->filename(), "rwc"sv));
-    TRY(Core::System::unveil("/dev/audio", "wc"));
-    TRY(Core::System::unveil(nullptr, nullptr));
+    // TRY(Core::System::unveil(config->filename(), "rwc"sv));
+    // TRY(Core::System::unveil("/dev/audio", "wc"));
+    // TRY(Core::System::unveil(nullptr, nullptr));
```

Comment:
Commented out code

Reply:
Hmmm, I might have forgotten to test this with the unveils set properly. There were some extremely weird crashes as soon as I introduced the second IPC server, but I'm not sure if that is related.